WooCommerce/Classes/ViewRelated/ListSelector/PaginatedListSelectorViewController.swift

```diff
@@ -387,6 +389,12 @@ private extension PaginatedListSelectorViewController {
             onReload()
         }
 
-        try? resultsController.performFetch()
+        do {
+            try resultsController.performFetch()
+        } catch {
+            CrashLogging.logError(error)
```

Comment:
non-blocking thought: once we consolidate the error logging after https://github.com/woocommerce/woocommerce-ios/issues/2364, it'd be nice to use the crash logger in the app layer as well